libs/utils/logWebVitals.js

```diff
@@ -0,0 +1,97 @@
+const LANA_CLIENT_ID = 'pageperf';
+
+let lanaSent = false;
+
+function sendToLana(lanaData) {
+  if (lanaSent) return;
+  const ua = window.navigator.userAgent;
+
+  Object.assign(lanaData, {
+    chromeVer: (ua.match(/Chrome\/(\d+\.\d+\.\d+\.\d+)/) || [])[1] || '',
```

Comment:
Nit: This can be shorter with optional chaining.
```suggestion
    chromeVer: ua.match(/Chrome\/(\d+\.\d+\.\d+\.\d+)/)?.[1] || '',
```